haskell/src/Lorentz/Contracts/Stablecoin.hs

```diff
@@ -62,26 +61,34 @@ metadataRegistryContractPath = "./test/resources/metadata.tz"
 ------------------------------------------------------------------
 -- Parameter
 
-type ConfigureMinterParam =
-  ( "minter" :! Address
-  , ( "current_minting_allowance" :! Maybe Natural
-    , "new_minting_allowance" :! Natural
-    )
-  )
+data ConfigureMinterParam = ConfigureMinterParam
+ { cmpMinter :: Address
+ , cmpCurrentMintingAllowance :: Maybe Natural
+ , cmpNewMintingAllowance :: Natural
+ }
+ deriving stock Show
+
+instance Buildable ConfigureMinterParam where
+  build = genericF
 
-pattern ConfigureMinterParams :: Address -> Maybe Natural -> Natural -> ConfigureMinterParam
-pattern ConfigureMinterParams addr cma nma <-
-  ( arg #minter -> addr
-  , (arg #current_minting_allowance -> cma, arg #new_minting_allowance -> nma))
-{-# COMPLETE ConfigureMinterParams #-}
+deriving anyclass instance IsoValue ConfigureMinterParam
+deriving anyclass instance HasAnnotation ConfigureMinterParam
```

Comment:
Why does it have to be standalone btw?

Reply:
Otherwise, I get these errors:

```
• No instance for (WellTyped
                     (GValueType (Rep ConfigureMinterParam)))
    arising from the 'deriving' clause of a data type declaration
  Possible fix:
    use a standalone 'deriving instance' declaration,
      so you can specify the instance context yourself
• When deriving the instance for (IsoValue ConfigureMinterParam)
```
```
• No instance for (GHasAnnotation (Rep ConfigureMinterParam))
    arising from the 'deriving' clause of a data type declaration
  Possible fix:
    use a standalone 'deriving instance' declaration,
      so you can specify the instance context yourself
• When deriving the instance for (HasAnnotation
                                    ConfigureMinterParam)
```

It seems it's not able to find the `Generic` instance. Maybe it's because it's being generated using TH?

Reply:
Okay, if there is a compile error with normal deriving, I don't mind using standalone deriving, not a problem.

> Maybe it's because it's being generated using TH?

Yes, probably.